packages/backend/src/peer/errors.ts

```diff
@@ -1,6 +1,7 @@
 export enum PeerError {
   DuplicateIncomingToken = 'DuplicateIncomingToken',
   InvalidStaticIlpAddress = 'InvalidStaticIlpAddress',
+  InvalidHTTPEndpoint = 'InvalidHTTPEndpoint',
```

Comment:
This can be included in the `CreatePeer` resolver tests
https://github.com/interledger/rafiki/blob/5f34cc00d2c7c9a22e4b28266a84a61f351f87d7/packages/backend/src/graphql/resolvers/peer.test.ts#L143-L148